.travis.yml

```diff
@@ -1,7 +1,10 @@
 ---
 language: node_js
 node_js:
+  - "4"
   - "6"
+  - "7"
+  - "8"
```

Comment:
This setup makes the test suite really huge (and slow!). This gives us numberOfNodeVersions * numberOfEmbyTryScenarios test jobs. Do we really need this, or would testing with node 4 not be enough? 

A setup were we test the different node versions only for one ember-try scenario, and run all scenarios with e.g. node 6 would be nice I think, but no idea how to do this in Travis...